core-api/src/main/java/com/optimizely/ab/config/parser/JsonConfigParser.java

```diff
@@ -266,19 +266,43 @@ public ProjectConfig parseProjectConfig(@Nonnull String json) throws ConfigParse
         List<Audience> audiences = new ArrayList<Audience>(audienceJson.length());
 
         for (Object obj : audienceJson) {
-            JSONObject audienceObject = (JSONObject)obj;
+            JSONObject audienceObject = (JSONObject) obj;
             String id = audienceObject.getString("id");
             String key = audienceObject.getString("name");
             String conditionString = audienceObject.getString("conditions");
-
-            JSONArray conditionJson = new JSONArray(conditionString);
-            Condition conditions = parseConditions(conditionJson);
+            Condition conditions;
+            if (conditionString.startsWith("[")) {
+                JSONArray conditionJsonArray = new JSONArray(conditionString);
+                conditions = parseConditions(conditionJsonArray);
+            } else {
+                JSONObject conditionJsonObject = new JSONObject(conditionString);
+                conditions = parseConditions(conditionJsonObject);
+            }
             audiences.add(new Audience(id, key, conditions));
         }
 
         return audiences;
     }
 
+    private Condition parseConditions(JSONObject conditionJson) {
+        List<Condition> conditions = new ArrayList<Condition>();
+
+        JSONObject conditionMap = conditionJson;
+        String value = null;
+        if (conditionMap.has("value")) {
+            value = conditionMap.getString("value");
+        }
+        conditions.add(new UserAttribute(
+                (String) conditionMap.get("name"),
```

Comment:
we use `conditionMap.getString(...)` few lines above, but `(String) conditionMap.get(...)` here. Are those intentionally different?

Side-note: majority of the time, `foo.toString()` is better than `(String) foo`